meson.build

```diff
@@ -0,0 +1,112 @@
+project(
+    'pyconcrete',
+    'c',
+    meson_version: '>= 1.1',
+)
+
+INSTALL_DIR = 'pyconcrete'
+PYCONCRETE_EXE_NAME = 'pyconcrete'
+BUILD_UTILITY_DIR = './meson_utility/'
+
+version = run_command('cat', 'VERSION', check: true).stdout().strip()
```

Comment:
Non-portable, unix-only. Meson has this as a builtin instead: https://mesonbuild.com/Fs-module.html#read

However what you actually want to do is add this directly to the project setup:

```meson
project(
    'pyconcrete',
    'c',
    meson_version: '>= 1.1',
    version: files('VERSION'),
)
```
meson will directly read this in and you can access the result as `meson.project_version()`. You can then list it in pyproject.toml as `dynamic = ["version"]` and meson-python will retrieve it from here and set it for the wheel.

This means you don't need to bump the version number in both the VERSION file and pyproject.toml

Reply:
It's a great solution to bump version at the same file. Thanks

Reply:
addressed 0e6b5158363b387bed7293a5ab8d36760fdc4e39